arc/src/arc_curie_pme.c

```diff
@@ -232,7 +236,8 @@ u16_t curie_pme_get_committed_count(void)
     return (getFORGET_NCOUNT() & 0xff);
 }
 
-PATTERN_MATCHING_CLASSIFICATION_MODE curie_pme_get_classifier_mode(void) // RBF or KNN
+PATTERN_MATCHING_CLASSIFICATION_MODE
+curie_pme_get_classifier_mode(void)  // RBF or KNN
```

Comment:
Again I'd rather move the comment up with more text and/or create a shortcut name for this return type.